core/capabilities/webapi/target/types.go

```diff
@@ -0,0 +1,19 @@
+package target
+
+const (
+	AllAtOnce  string = "AllAtOnce"
+	RoundRobin string = "RoundRobin"
+)
+
+type WorkflowInput struct {
+	URL     string            `json:"url"`               // URL to query, only http and https protocols are supported.
+	Method  string            `json:"method,omitempty"`  // HTTP verb, defaults to GET.
+	Headers map[string]string `json:"headers,omitempty"` // HTTP headers, defaults to empty.
+	Body    string            `json:"body,omitempty"`    // Base64-encoded binary body, defaults to empty.
```

Comment:
Note: can we just use bytes here? Why are we expecting the user to encode the body ahead of time; that seems to leak implementation details.

Reply:
I thought `mapstructure` used inside `Map.UnwrapTo` can't deserialize to []byte here. Will test it out.